website/blog/2018-06-19-single-spa-parcels-explained.md

```diff
@@ -0,0 +1,70 @@
+---
+title: single-spa parcels, explained
+author: Joel Denning
+authorURL: https://twitter.com/joelbdenning
+authorImageURL: https://avatars2.githubusercontent.com/u/5524384?s=400&u=ff145fcb2ae5305555628a446e9f725d4e145aaa&v=4
+---
+Ever since single-spa@1.0.0, the single-spa team has been dedicated to bringing microservices to the frontend. We have made it possible for AngularJS, React, Angular, Vue, and other frameworks to coexist side by side in the same page.
+
+And with the release of [version 4](https://github.com/CanopyTax/single-spa/releases/tag/v4.0.0), I’m pleased to announce that [single-spa](https://github.com/CanopyTax/single-spa) is expanding that effort so that ***individual components*** written with different frameworks can interoperate. It is new terrain for the single-spa community, which previously had focused on getting large applications to interoperate with each other, instead of the individual components.
```

Comment:
Should we link to single-spa.js.org here?

Reply:
Well if they're already on the blog that means they're already on single-spa.js.org, so linking to it seems sorta redundant. I am thinking a link to the github page might be nice for people who are looking for the code